packages/cursorless-vscode-e2e/src/suite/scopeProvider/runCustomRegexScopeInfoTest.ts

```diff
@@ -26,30 +28,17 @@ export async function runCustomRegexScopeInfoTest() {
   try {
     await assertCalledWithoutScopeInfo(fake, scopeType);
 
-    await writeFile(
-      cursorlessTalonStateJsonPath,
-      JSON.stringify(spokenFormJsonContents),
-    );
+    talonSpokenForms.mockSpokenFormEntries(spokenFormEntries);
     await assertCalledWithScopeInfo(fake, unsupported);
 
     await openNewEditor(contents);
+    await sleep(10);
```

Comment:
Added a short sleep to just make sure that the document open event had triggered which is one of the events the scope provider uses to update available scopes.

Reply:
I would document that in a comment above the sleep, since the PR comment will be lost to time